string chess_move
string is_capture
string piece_to_move
string captured_piece